wadsrc/static/zscript/shared/sharedmisc.txt

```diff
@@ -159,7 +159,7 @@ class CustomSprite : Actor
 		String name;
 		Super.BeginPlay ();
 
-		format(name, "BTIL%04d", args[0] & 0xffff);
+		//format(name, "BTIL%04d", args[0] & 0xffff); // note: this does NOTHING and did NOTHING. what?
 		picnum = TexMan.CheckForTexture (name, TexMan.TYPE_Build);
```

Comment:
Side note: with UDMF, pretty sure args[0] can reach 5 digits even with clamping to WORD.